EventListener/BugsnagListener.php

```diff
@@ -56,12 +58,17 @@ public function __construct(Client $client, SymfonyResolver $resolver, $auto)
     /**
      * Handle an incoming request.
      *
-     * @param \Symfony\Component\HttpKernel\Event\GetResponseEvent $event
+     * @param GetResponseEvent|RequestEvent $event
      *
      * @return void
      */
-    public function onKernelRequest(GetResponseEvent $event)
+    public function onKernelRequest($event)
     {
+        // Compatibility with Symfony < 5 and Symfony >=5
+        if (!$event instanceof GetResponseEvent && !$event instanceof RequestEvent) {
+            return;
```

Comment:
After reading [this comment](https://github.com/nelmio/NelmioSecurityBundle/pull/211#discussion_r353064083), I think I should have added that in the initial PR (ie throwing a `InvalidArgumentException` in each listener method). What do you think?